repository/error/error.go

```diff
@@ -0,0 +1,54 @@
+package error
+
+import (
+	"errors"
+
+	svcerr "release-manager/service/error"
+
+	postgrestgo "github.com/nedpals/postgrest-go/pkg"
+	"github.com/nedpals/supabase-go"
+)
+
+var (
+	PostgresResourceNotFoundCode = "PGRST116"
+)
+
+var (
+	ErrMultipleOrNoRecordsReturned = errors.New("supabase: one record expected, multiple (or no) rows returned")
+)
+
+func ToSvcErrFromAuth(err error) error {
+	var supabaseErr *supabase.ErrorResponse
+	if errors.As(err, &supabaseErr) {
+		switch supabaseErr.Code {
+		case 401, 404:
```

Comment:
In case of Auth - both 401, 404 errors means authentication failed (token is not valid or user does not exist anymore)